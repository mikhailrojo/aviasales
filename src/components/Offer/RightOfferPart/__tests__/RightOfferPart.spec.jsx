import * as React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import {RightOfferPart} from '../index';

const TEST_OFFER = {
	arrival_date: "12.05.18",
	arrival_time: "22:10",
	carrier: "TK",
	departure_date: "12.05.18",
	departure_time: "16:20",
	destination: "TLV",
	destination_name: "Тель-Авив",
	origin: "VVO",
	origin_name: "Владивосток",
	price: 12400,
	stops: 3,
};

describe('Snapshot', () => {
	it('render RightOfferPart with 1 offer', () => {
		const renderer = new ShallowRenderer();
		const component = renderer.render(
			<RightOfferPart
				{...TEST_OFFER}
			/>
		);
		expect(component).toMatchSnapshot();
	});
});