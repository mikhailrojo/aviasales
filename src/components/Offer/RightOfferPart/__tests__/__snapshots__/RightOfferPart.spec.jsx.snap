// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot render RightOfferPart with 1 offer 1`] = `
<styled.div>
  <div>
    <styled.div>
      16:20
    </styled.div>
    <styled.div>
      VVO Владивосток
    </styled.div>
    <styled.div>
      12 mai 2018, Sam.
    </styled.div>
  </div>
  <styled.div>
    3 пересадки
    <styled.div>
      <styled.div />
      <styled.div />
    </styled.div>
  </styled.div>
  <div>
    <styled.div>
      22:10
    </styled.div>
    <styled.div>
      Тель-Авив TLV
    </styled.div>
    <styled.div>
      12 mai 2018, Sam.
    </styled.div>
  </div>
</styled.div>
`;
